.travis.yml

```diff
@@ -229,7 +229,8 @@ install:
       return 0
     }
 script:
-  - export CMAKE_ARGS="-DLUA_LIBRARY=${LUALIBRARY} -DLUA_INCLUDE_DIR=${LUAINCLUDE} -D OVERRIDE_VERSION=$AWESOME_VERSION -DSTRICT_TESTS=true -D DO_COVERAGE=$DO_COVERAGE -D CMAKE_C_FLAGS=-Werror"
+  - if [ "$MANUAL_SCREENS" != "1" ]; then export MANUAL_SCREENS=0; fi
+  - export CMAKE_ARGS="-DLUA_LIBRARY=${LUALIBRARY} -D LUA_INCLUDE_DIR=${LUAINCLUDE} -D OVERRIDE_VERSION=$AWESOME_VERSION -D STRICT_TESTS=true -D DO_COVERAGE=$DO_COVERAGE -D TEST_MANUAL_SCREENS=$MANUAL_SCREENS -D CMAKE_C_FLAGS=-Werror"
```

Comment:
This could also look like:
```
￼- if [  "$MANUAL_SCREENS" = "1" ]; then CMAKE_ARGS="$CMAKE_ARGS -D TEST_MANUAL_SCREENS=1"; fi
```

I.e. keep the original "export CMAKE_ARGS" (but OK to fix whitespace there maybe still), and then extend it for MANUAL_SCREENS=1.